src/e2e-test/java/io/cdap/plugin/odp/locators/ODPLocators.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.odp.locators;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+import java.util.List;
+
+/**
+ * ODPLocators.
+ */
+public class ODPLocators {
+
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'SAP ODP')]")
+  public static WebElement sapODPSource;
+
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'-SNAPSHOT')]")
```

Comment:
can this locator be made more specific so that it is used to locate properties of SAP ODP plugin specifically?because there can be other plugins as well which can match this.

Reply:
done